docs/docs/admin/monitoring/logging/explorer.md

```diff
@@ -0,0 +1,119 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Grafana Explorer
```

Comment:
The sidebar name being "Log Explorer" and the title being "Grafana Explorer" is a bit confusing.

Reply:
Changed it to consistent naming: Grafana Log Explorer